test_files/declare.sickle.ts

```diff
@@ -1,5 +1,17 @@
-declare module DeclareTest {
+declare module DeclareTestModule {
   export class Foo {
     constructor();
   }
 }
+
+declare interface DeclareTestInterface {
```

Comment:
This is pretty far away from what a Closure user would write. I think it'd be more idiomatic to at least use an @interface with @record here, but I guess we cannot do that?

I'm fine with this, but I guess at some point we need infrastructure that let's us generate truly arbitrary JS code. If only in an "overflow buffer" that we later append to the final .js file before writing it.